packages/ckeditor5-table-resize/tests/tablecolumnresizer/utils.js

```diff
@@ -330,6 +338,131 @@ describe( 'TableColumnResize utils', () => {
 				expect( affectedTables.size ).to.equal( 0 );
 			} );
 		} );
+
+		describe( 'should not crash if no affected table has been found', () => {
+			let model, differ, editor, editorElement, linkCommand, unlinkCommand, highlightCommand;
+
+			beforeEach( async () => {
+				editorElement = document.createElement( 'div' );
+				document.body.appendChild( editorElement );
+
+				editor = await ClassicEditor.create( editorElement, Object.assign( {}, {
+					plugins: [ Table, Paragraph, LinkEditing, HighlightEditing, Bold ]
+				} ) );
+
+				model = editor.model;
+				differ = model.document.differ;
+				linkCommand = new LinkCommand( editor );
+				unlinkCommand = new UnlinkCommand( editor );
+				highlightCommand = new HighlightCommand( editor );
+			} );
+
+			afterEach( async () => {
+				editorElement.remove();
+
+				await editor.destroy();
+			} );
+
+			it( 'when link is being removed', () => {
+				setData( model,
+					'<table columnWidths="20%,25%,55%">' +
+						'<tableRow>' +
+							'<tableCell columnIndex="0">' +
+								'<paragraph>00[foo]</paragraph>' +
+							'</tableCell>' +
+						'</tableRow>' +
+					'</table>'
+				);
+
+				expect( linkCommand.value ).to.be.undefined;
+
+				linkCommand.execute( 'url' );
+
+				expect( linkCommand.value ).to.be.equal( 'url' );
+
+				unlinkCommand.execute();
```

Comment:
Correct me if I'm wrong, but I think we are only interested in the second part of the test - removing the attribute - so we can simplify the test by applying the necessary markup directly to the model. Same for the other tests.